lib/entryFromResponse.js

```diff
@@ -79,8 +79,9 @@ module.exports = function(entry, response, page, options) {
     status: response.status,
     statusText: response.statusText,
     content: {
+      encodig: response.encoding,
```

Comment:
This would be really useful to have!
May I suggest to have the property corrected to `encoding`?

Reply:
See https://github.com/beatchristen/chrome-har/commit/20b078ef0d720df8b784c906faa555b203b0097b for my variant, which also tries to adhere to the 'optionality' aspect of the property.